tests/testsearch.py

```diff
@@ -39,6 +40,15 @@ class SearchTests(unittest.TestCase):
     def setUp(self):
         self.__startTime = time.time()
         logger.info("Starting %s at %s", self.id(), time.strftime("%Y %m %d %H:%M:%S", time.localtime()))
+        HERE = os.path.abspath(os.path.dirname(__file__))
+        self.__dirPath = os.path.join(HERE, "files")
+        self.__4hhbBcif = os.path.join(self.__dirPath, "4hhb.bcif")
+        self.__4hhbCif = os.path.join(self.__dirPath, "4hhb.cif")
+        self.__4hhbPdb = os.path.join(self.__dirPath, "4hhb.pdb")
+        self.__7n0rPdbGz = os.path.join(self.__dirPath, "7n0r.pdb.gz")
+        self.__7n0rCifGz = os.path.join(self.__dirPath, "7n0r.cif.gz")
+        self.__invalidTxt = os.path.join(self.__dirPath, "invalid.txt")
+        self.__4hhbAssembly1 = os.path.join(self.__dirPath, "4hhb-assembly1.cif.gz")
```

Comment:
Although not critical, I suggest including [4HHB.pdb1.gz](https://files.rcsb.org/download/4HHB.pdb1.gz).